lib/truffle/rbconfig-for-mkmf.rb

```diff
@@ -39,12 +39,12 @@
 
 opt_passes = ['-always-inline', '-mem2reg', '-constprop'].join(' ')
 linkflags = [
-  '-g',                              # Show debug information such as line numbers in backtrace
+  '-g', # Show debug information such as line numbers in backtrace
   '-Wimplicit-function-declaration', # To make missing C ext functions clear
-  '-Wno-unknown-warning-option',     # If we're on an earlier version of clang without a warning option, ignore it
-  '-Wno-int-conversion',             # MRI has VALUE defined as long while we have it as void*
-  '-Wno-int-to-pointer-cast',        # Same as above
-  '-Wno-unused-value',               # RB_GC_GUARD leaves
+  '-Wno-unknown-warning-option', # If we're on an earlier version of clang without a warning option, ignore it
+  '-Wno-int-conversion', # MRI has VALUE defined as long while we have it as void*
+  '-Wno-int-to-pointer-cast', # Same as above
+  '-Wno-unused-value', # RB_GC_GUARD leaves
   '-Wno-incompatible-pointer-types', # Fix byebug 8.2.1 compile (st_data_t error)
```

Comment:
This one might look weird, but now most lines fit within 80 characters.

Reply:
Having to fit in 80 chars seems like a 4 decade old legacy that practically doesn't matter any longer.

Reply:
80 chars is arbitrary, but it means most of the lines fit on half a screen, vs almost none.

Reply:
FWIW, half of my screen with Atom displays around 116 chars.

Reply:
Let's cap the file length to 25 lines so it all fits vertically, too ;-)

In all seriousness, I think it's good to have an upper-bound, but I think it ought to be a soft-limit. Arbitrary line breaks can harm legibility just as much. Or, in this case, changing the alignment of the comments harms legibility to me.

Reply:
Yeah, I just meant here there is the nice side-effect to reduce line length significantly, which can be useful at times.
But I agree here it looks nicer aligned. Alternatively, we could have alternating lines of comments and elements to avoid long lines.